src/base/QXmppJingleIq.h

```diff
@@ -157,6 +159,79 @@ class QXMPP_EXPORT QXmppJingleIq : public QXmppIq
         TransportReplace
     };
 
+    ///
+    /// Party muting or unmuting itself
+    ///
+    /// \since QXmpp 1.5
+    ///
+    enum RtpSessionStateMutingCreator {
```

Comment:
```suggestion
    enum Creator {
```
I don't know whether this is actually useful, but what do you think about using this enum for general purposes and calling it just `Creator`?

Reply:
I replaced it by a bool.

Reply:
please use an enum, that's more explicit

Reply:
I only renamed it now.